native/cocos/bindings/event/EventDispatcher.h

```diff
@@ -276,7 +269,8 @@ class EventDispatcher {
     static void     dispatchCustomEvent(const CustomEvent &event);
 
 private:
-    static void doDispatchEvent(const char *eventName, const char *jsFunctionName, const std::vector<se::Value> &args);
+    static void doDispatchJsEvent(const char *jsFunctionName, const std::vector<se::Value> &args);
+    static void doDispatchCustomEvent(const char *eventName, int argNum, ...);
```

Comment:
Does this function only sent event to cpp codes, not sent to JS? If so, please use a more meaningful name.

Reply:
change to doDispatchNativeEvent, "CustomEvent" means actually nothing here